src/docs/sphinx/quickstart.rst

```diff
@@ -267,28 +267,268 @@ must be specified using either:
+      externals:
+      - spec: perl@v5.34.1
+         prefix: /opt/local
+   pkg-config:
+      buildable: false
+      externals:
+      - spec: pkg-config@0.29.2
+         prefix: /opt/local
+   tar:
+      buildable: false
+      externals:
+      - spec: tar@3.3.2
+         prefix: /usr
+   readline:
+      buildable: false
+      externals:
+      - spec: readline@8.1.2.000
+         prefix: /opt/local
+   unzip:
+      buildable: false
+      externals:
+      - spec: unzip@6.0
+         prefix: /usr
+   zlib:
+      buildable: false
+      externals:
+      - spec: zlib@1.2.12
+         prefix: /opt/local
+
+Notes:
+
+* OpenGL is not supported on modern Macs, hence those packages are not specified as they are on other platforms. 
+  This means that the optional ``glvis`` spec can't be built.
+* The version specs should be set to the actual versions of the packages you have, which will not neccesarily be the same as the above.
+
+The above spack settings and MacPorts packages will cover the basic installation of Serac. 
+If you want to build the optional devtools, you should install the additional packages with MacPorts:
+
+* cmake (CMake is present by default on current MacOS distributions, but is often out of date, hence we recommend installing this package.)
+* cppcheck
+* doxygen
+
+Then, append the following to ``packages.yaml``:
+
+.. code-block:: yaml
+
+   cmake:
+    version: [3.22.4]
+    buildable: false
+    externals:
+    - spec: cmake@3.22.4
+      prefix: /opt/local
+  cppcheck:
+    version: [2.3]
+    buildable: false
+    externals:
+    - spec: cppcheck@2.3
+      prefix: /usr/local
+  doxygen:
+    version: [1.8.13]
+    buildable: false
+    externals:
+    - spec: doxygen@1.8.13
+      prefix: /usr/local
+  llvm:
+    version: [10.0.0]
+    buildable: false
+    externals:
+    - spec: llvm+clang@10.0.0
+      prefix: <path/to/llvm/10>
+  python:
+    buildable: false
+    externals:
+    - spec: python@3.9
+      prefix: <path/to/python/venv>
+
+Notes:
+
+* LLVM/Clang is needed for the style check tools. 
+  The *exact* version 10.0.0 is apparently highly recommended, since other versions may format the code slightly differently,
+  which will mean that pull requests formatted with them may trigger style errors in the CI checks.
+* The placeholders ``<path/to/llvm/10>`` and ``<path/to/python/venv>`` need to be filled in with actual paths.
+  See the following two notes. 
+* LLVM 10.0.0 has been superseded as the version for llvm-10; so this package is not easily installable with MacPorts.
+  You must build it yourself and then point to the build location.
+* For ``<path/to/python/venv>``, specify the the virtual environment directory.
+
+Building dependencies
+^^^^^^^^^^^^^^^^^^^^^
+
+The invocation of ``uberenv.py`` is slightly modified from the standard instructions above: 
+
+.. code-block:: bash
+
+   $ ./scripts/uberenv/uberenv.py --prefix=../path/for/TPLs --spec="clang@12.0.1 ^openmpi@4.1.4"
```

Comment:
should be `./scripts/uberenv/uberenv.py --prefix=../path/for/TPLs --spec="%clang@12.0.1 ^openmpi@4.1.4"` (needed `%` by clang)

Reply:
Good catch. Fixed.